README.md

```diff
@@ -57,15 +57,17 @@ Then add the following dependencies in your app `build.gradle` or `build.gradle.
 **Groovy**
 
 ```groovy
-debugImplementation "com.infinum.sentinel:sentinel:1.3.1"
-releaseImplementation "com.infinum.sentinel:sentinel-no-op:1.3.1"
+def sentinelVersion = "1.3.2"
```

Comment:
As the project already has 2 versions defined, would be good to go in a different direction than this and use a single version everywhere. On some libraries like [PrinceOfVersions](https://github.com/infinum/Android-Prince-of-Versions/blob/master/tasks.gradle#L20) we have a gradle task to update readme with the current version.

Reply:
I will add idea for Gradle task to the list